webextension/background.js

```diff
@@ -70,6 +70,18 @@ const assignManager = {
     }
   },
 
+  // We return here so the confirm page can load the tab when exempted
+  async _exemptTab(m) {
+    const pageUrl = m.pageUrl;
+    // If we have existing data and for some reason it hasn't been deleted etc lets update it
+    const siteSettings = await this.storageArea.get(pageUrl);
+    if (siteSettings) {
+      siteSettings.exempted.push(m.tabId);
+      await this.storageArea.set(pageUrl, siteSettings);
```

Comment:
This permanently stores the exemption for the tab ID, which means after a Firefox restart, a tab with the same ID going to the site will automatically skip the confirm page and open in the current container.